language/RELEASES.md

```diff
@@ -0,0 +1,72 @@
+* Fixed a compiler crash related to expressions with short-circuiting ([#7574](https://github.com/diem/diem/issues/7574)).
+* Fixed an incorrect code generation bug that could occur when a function parameter is assigned a new value exactly once in the function ([#7370](https://github.com/diem/diem/pull/7370)).
+* Fixed the bytecode source map mapping from local names to indexes so that function parameters go before locals ([#7371](https://github.com/diem/diem/pull/7371)).
+* Fixed a compiler crash when a struct is assigned without specifying its fields ([#7385](https://github.com/diem/diem/issues/7385)).
+* Fixed a compiler crash when attempting to put a `spec` block inside a `spec` context ([#7387](https://github.com/diem/diem/issues/7387)).
+* An integer literal value that is too large for its declared type will no longer cause a compiler crash ([#7388](https://github.com/diem/diem/issues/7388)).
+* Fixed a compiler crash caused by incorrect number of type parameters in pack/unpack expressions ([#7401](https://github.com/diem/diem/pull/7401)).
+* Module names and module members are now restricted from starting with underscores (‘_’) , which also avoids a crash ([#7572](https://github.com/diem/diem/issues/7572)).
+* Prover specifications are now included in the compiler’s dependency ordering calculation ([#7960](https://github.com/diem/diem/pull/7960)).
+* Modified the compiler optimization to remove fall-through jumps so that loop headers are not coalesced, which improves the prover’s ability to handle loop specifications ([#8049](https://github.com/diem/diem/pull/8049)).
+
+## Command Line Interpreter (CLI)
+
+The Move CLI has been enhanced in several ways:
+
+* The CLI now supports safe module republishing with checks for breaking changes ([#6753](https://github.com/diem/diem/pull/6753)).
+* Added a new `doctor` command to detect inconsistencies in storage ([#6971](https://github.com/diem/diem/pull/6971), [#7010](https://github.com/diem/diem/pull/7010), and [#7013](https://github.com/diem/diem/pull/7013)).
+* The `publish` command’s `—-dry-run` option has been removed ([#6957](https://github.com/diem/diem/pull/6957)). Use the equivalent "check" command instead.
+* The `test` command has a new `--create` option to create test scaffolding ([#6969](https://github.com/diem/diem/pull/6969)).
+* The verbose output now includes the number of bytes written ([#7757](https://github.com/diem/diem/pull/7757)).
+
+## Other Tools
+
+* Created a new bytecode-to-source explorer tool for Move ([#7508](https://github.com/diem/diem/pull/7508)).
+* The resource viewer can now be better used to traverse data structures because the fields of `AnnotatedMoveStruct` are no longer private and `AnnotatedMoveValue::Vector` preserves the type information for its elements ([#7166](https://github.com/diem/diem/pull/7166)).
+* The writeset-generator and transaction replay tools have been significantly enhanced to support the process of upgrading the Diem Framework.
```

Comment:
```suggestion
* The writeset-generator and transaction-replay tools have been significantly enhanced to support the process of upgrading the Diem Framework.
```

Reply:
I had intended to leave these tool names as generic terms and not the real names. For some reason, hyphenating "transaction-replay" makes it seem like the real name. After re-reading this bullet point, I'm now thinking to use the real names (i.e., with the `diem-` prefix) and quote them as code. Arguably these could also be listed in the Diem changelog or dropped altogether....